setup.py

```diff
@@ -1,21 +0,0 @@
-#!/usr/bin/env python
```

Comment:
@Vikram9866 doh, how did I miss this??  Why, why was this removed?  Please fix ASAP.